lightning/src/ln/channelmanager.rs

```diff
@@ -1105,8 +1105,8 @@ impl ChannelDetails {
 }
 
 /// Used by [`ChannelManager::list_pending_payments`] to express the status of payments that are
-/// currently pending resolution. These not only include payments that are currently inflight, but
-/// also those that may have yet to find a successful path or have outstanding HTLCs.
+/// currently pending. These not only include payments that are currently inflight, but also those
+/// that may have yet to find a successful path or have unresolved HTLCs.
```

Comment:
Seems that "payments that are currently inflight" is defined by what follows, so saying "not only" and "but also" isn't accurate. What suggest dropping those words and replacing the later simply with "i.e.,".

Reply:
I just made it all one line: "These include payments that have yet to find a successful path, or have unresolved HTLCs."